Building-ChakraCore.md

```diff
@@ -113,3 +113,21 @@ Some details on build script arguments used above:
 * ```--icu=..``` specify the location of ICU library (use `--no-icu` if you don't need unicode support)
 * ```--test-build``` enable command line options and some optimizations
 * ```-j=2``` compile with 2 threads
+
+### Building and debugging in XCode ###
+
+Generate an XCode project:
+* ```./build.sh --xcode --icu=/usr/local/opt/icu4c/include --test-build --static```
```

Comment:
can we mention`--debug` too ? (in addition `--test-build`) IMHO, `debug` build is more useful with xcode.

Reply:
Sure, should I just swap out `--test-build` with `--debug`? I don't really know the difference.

Reply:
could be nice. In that case you don't need the last part of about (xcode optimization settings) because it's `-O0` for debug builds.

Reply:
Oooh, `--debug` is so much better then! I'll update the PR.